component.yaml

```
@@ -0,0 +1,39 @@
+platform: Common
+buildPlatform: ReactNative
+displayName: "React Native UI Test"
+description: "Run React Native UI Tests with Detox"
+inputs:
+  - key: "AC_REPOSITORY_DIR"
+    defaultValue: "$AC_REPOSITORY_DIR"
+    isRequired: true
+    title: Repository Directory
+    description: Specifies the cloned repository directory.
+    helpText:
+  - key: "AC_OUTPUT_DIR"
+    defaultValue: "$AC_OUTPUT_DIR"
+    isRequired: true
+    title: Output Direction
+    description: Specify the output directory.
+    helpText:
+  - key: "AC_RN_DETOX_CONFIGURATION"
+    defaultValue: "$AC_RN_DETOX_CONFIGURATION"
+    isRequired: true
+    title: Detox Configuration Name
+    description: Specify the Detox configuration name to used when building and running the tests.
+    helpText:
+  - key: "AC_RN_DETOX_ARGS"
```

IMO it is more clear:
```suggestion
  - key: "AC_RN_DETOX_TEST_ARGS"
```